document-store/src/main/java/org/hypertrace/core/documentstore/Collection.java

```diff
@@ -127,6 +128,16 @@ public interface Collection {
    */
   CloseableIterator<Document> aggregate(final org.hypertrace.core.documentstore.query.Query query);
 
+  /**
+   * Query the documents conforming to the query specification.
+   *
+   * @param query The query specification
+   * @param queryOptions The query options
+   * @return {@link CloseableIterator} of matching documents
+   */
+  CloseableIterator<Document> query(
```

Comment:
Q: the existing query api deprecated?

Reply:
Don't think that's required. If the clients want to query with the default configurations, the older method can be used.